id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/utils/IdentifiableTest.scala

```diff
@@ -16,4 +16,15 @@ class IdentifiableTest extends FunSpec with PropertyChecks with Matchers {
     }
   }
 
+  // We have a rule that identifiers can't start with a number, as this
+  // makes them awkward to use as a string in certain contexts.
+  it("should never generate an identifier that starts with a number") {
+    forAll(minSuccessful(100)) { (_: Int) =>
+      {
+        val id = Identifiable.generate
+        assert(!id.head.isDigit)
```

Comment:
👍

I’ve taken a habit from Python – you generally refer to boolean results explicitly – but you’re right, consistency with the existing code should win out here.